pvacseq/lib/input_file_converter.py

```diff
@@ -290,3 +297,88 @@ def execute(self):
 
         writer.close()
         reader.close()
+
+class IntegrateConverter(InputFileConverter):
+    def input_fieldnames(self):
+        return [
+            'chr 5p',
+            'start 5p',
+            'end 5p',
+            'chr 3p',
+            'start 3p',
+            'end 3p',
+            'name of fusion',
+            'tier of fusion',
+            'strand 5p',
+            'strand 3p',
+            'quantitation',
+            'is canonical boundary',
+            'can be in-frame',
+            'peptides',
+            'fusion positions',
+            'number of nucleotides in the break',
+            'transcripts',
+            'is canonical intron dinucleotide',
+        ]
+
+    def execute(self):
+        reader = open(self.input_file, 'r')
+        csv_reader = csv.DictReader(reader, delimiter='\t', fieldnames=self.input_fieldnames())
+        writer = open(self.output_file, 'w')
+        tsv_writer = csv.DictWriter(writer, delimiter='\t', fieldnames=self.output_headers())
+        tsv_writer.writeheader()
+        transcript_count = {}
+        for entry in csv_reader:
+            output_row = {
+                'chromosome_name'            : entry['chr 5p'],
+                'start'                      : entry['start 5p'],
+                'stop'                       : entry['end 5p'],
+                'reference'                  : 'fusion',
+                'variant'                    : 'fusion',
+                'gene_name'                  : entry['name of fusion'],
+                'amino_acid_change'          : 'NA',
+                'ensembl_gene_id'            : 'NA',
+                'protein_position'           : 'NA',
+                'amino_acid_change'          : 'NA',
+                'transcript_expression'      : 'NA',
+                'gene_expression'            : 'NA',
+                'normal_depth'               : 'NA',
+                'normal_vaf'                 : 'NA',
+                'tdna_depth'                 : 'NA',
+                'tdna_vaf'                   : 'NA',
+                'trna_depth'                 : 'NA',
+                'trna_vaf'                   : 'NA',
```

Comment:
Yes, we used trna in the intermediate snv/indel tsv as well. In the final report output this is is called `Tumor RNA ..`. However, the command line threshold parameters for the coverage filter cutoff is also `trna-vaf` etc.